lib/game.ex

```diff
@@ -124,4 +128,128 @@ defmodule Game do
   def get_marker(game), do: game.player_2_mark
 
   def update_board(board, game), do: %{game | board: board}
+
+  def next_open(game), do: List.first(get_available_moves(game))
+
+  def handle_next_move(game) do
+    move = next_open(game)
+    player_move_list = determine_move_list(game)
+    game = update_available_moves(move, game)
+    game = update_player_move_list(game, player_move_list, move)
+    place_marker(move, game)
+  end
+
+  def best_move(game) do
+    # best_score = -100
+    # new_move = nil
+    game = assign_maximizer(game)
+
+    # Iterate through every cell
+    score_move_list =
+      Enum.map([1, 2, 3, 4, 5, 6, 7, 8, 9], fn move ->
+        IO.puts("\r\n\r\nTrying == #{move} ")
+
+        # Grab that move number in case we want to use it
+        IO.puts("AVAIL MOVES BELOW")
+        IO.inspect(game.available_moves)
+        IO.puts("AVAIL MOVES ABOVE")
+
+        IO.puts("MOVE IN AVAIL MOVES?")
+        IO.inspect(move in game.available_moves)
+        IO.puts("MOVE IN AVAIL MOVES?")
+
+        # If move is in available moves continue
+        if move in game.available_moves do
+          # IO.puts("BEST SCORE BEFORE ==== #{best_score}")
+          IO.puts("DOING == #{move} ")
+          # play the next available move
+          game = handle_move(move, game)
+          game = check_for_win(game)
+          {move, score(game)}
+
+        else
```

Comment:
by skip you mean return `nil`? 

```elixir
res = Enum.map([0,1,2,3,4,5], fn x ->
  if x in [0,2,4] do
    x
  end
end)

IO.inspect(res) # [0, nil, 2, nil, 4, nil]
```

Reply:
I have a filter, but I was having an issue in just skipping these. This is a syntax issue that I was fighting.